resources/metadata/detectExecuteScan.yaml

```
@@ -664,6 +664,17 @@ spec:
           - STAGES
           - STEPS
         default: false
+      - name: useDetect9
+        description:
+          "This flag enables the use of the supported version 9 of the Detect Script instead of v8"
```

```suggestion
          "This flag enables the use of the supported version 9 of the Detect Script instead of v10"
```

Default is v10 now, please also adjust comment on `useDetect8`

---

@CCFenner Existing comment is right as we are moving from lower to upper detect version, so  the comment "This flag enables the use of the supported version 9 of the Detect Script instead of v8" should be fine.

---

Maybe we should just refer to it as "the default" instead of a specific version.. 

This flag will only trigger v9 if v8 is not set, so it will overwrite the use ov v10.

---

I have changed the description now, default version is 10 now and user may use the detect 8 or detect 9 by setting particular field as True. I hope this is fine.

---

You added a new parameter, that is however not used, I would asked to remove that please.